Userland/Applications/SoundPlayer/PlaybackManager.cpp

```diff
@@ -46,7 +46,8 @@ void PlaybackManager::stop()
     m_current_buffer = nullptr;
 
     if (m_loader)
-        m_loader->reset();
+        [[maybe_unused]]
+        auto result = m_loader->reset();
```

Comment:
you can cast to void here to achieve the same "Result"

Reply:
*get out*

(yep, that's cleaner)